examples/Nat/TcpState.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using PacketDotNet;
+
+namespace Pax.Examples.Nat
+{
+  internal class TcpState : ITransportState<TcpPacket>
+  {
+    private TcpDirectionalState InOutConnection = TcpDirectionalState.None;
```

Comment:
Could you add a couple of lines of comments on the role of `InOutConnection` and `OutInConnection` please. I what you're doing makes sense -- you seem to be modelling the TCP machines at either end of the connection -- and it's worth confirming this in comments. Thanks!